fr/common.json

```
@@ -0,0 +1,452 @@
+        "isSubItemsVisible": "Montrer les objets intermédiaires",
+        "bestRoutes": "Recommandé",
+        "otherRoutes": "Autres"
+      }
+    },
+    "recommendation": {
+      "foods": "Nourriture",
+      "beverages": "Boisson",
+      "traps": "Piège",
+      "armors": "Objets fabricables sur cette route",
+      "armorsDescription": "Configurez votre Itinéraire et cliquez sur les éléments ci-dessous pour les équiper. Choisir  unarticle de la même catégorie remplacera l'ancien.",
+      "defaultContent": "Créez votre trajet"
+    },
+    "craftBoard": {
+      "title": "Informations de l'Itinéraire",
+      "showDetail": "Afficher Détails",
+      "equipped": "Équipement",
+      "inventory": "Inventaire",
+      "search": "Chercher Matériaux",
+      "gather": "Matériaux de Récolte/Chasse",
+      "tooltip": {
+        "inventory": "Seuls les objets nécessaires sont pris en compte.",
+        "inventoryOver": "Plus de dix objets sont nécessaires à cet instant"
+      }
+    },
+    "mapBoard": {
+      "options": {
+        "showGatherItem": {
+          "title": "Visibilité des matériaux de recherche",
+          "none": "Aucun",
+          "onlyMe": "Pour Moi",
+          "team": "Tous"
+        },
+        "routeHighlight": {
+          "title": "Montrer Trajet",
+          "onlyMe": "Pour Moi",
+          "team": "Tous"
+        },
+        "showAreaItemCount": {
+          "title": "Nombre total d'objets",
+          "on": "On",
+          "off": "Off"
+        }
+      }
+    }
+  },
+  "characters": {
+    "title": "Personnages",
+    "titleWithCharacter": "{{ queueType }} {{ character }} - Personnages",
+    "description": "Statistiques des Personnages d'Eternal Return, y compris le niveau, les matchups, l'ordre des compétences, les Itinéraires, les préférences d'objets, le taux de victoire, etc.",
+    "descriptionWithCharacter": "{{ queueType }} {{ character }} statistiques incluant le rang, les matchups, l'ordre des compétences, les Itinéraires, les préférences d'objets, le taux de victoire, etc.",
+    "overall": "Général",
+    "weaponAndCharacter": "Personnage / Arme",
+    "statisticsRange": "Portée",
+    "statisticsRange3Days": "Parties classées dans les trois derniers jours",
+    "statisticsRangeGameVersion": "Parties classés en {{ gameVersion }}",
+    "lastUpdate": "Dernière Mise à jour",
+    "moreButton": "Plus",
+    "subPages": {
+      "overview": "Aperçu",
+      "matchup": "Matchups",
+      "metrics": "Données",
+      "metricsTrend": "Tendance des données",
+      "route": "Itinéraire & Objets",
+      "skill": "Ordre des compétences"
+    },
+    "tierOptions": {
+      "t": "TOP 1000",
+      "a": "Tout",
+      "u": "Platine et Plus",
+      "l": "Or et Moins"
+    },
+    "range": {
+      "3d": "3 derniers Jours",
+      "version": "{{version}}"
+    },
+    "matchup": {
+      "centerDescription": "Ratio de Personnage / Kill ",
```

You used `éliminations` for `Élim. Moy.`, but in this field it's still `Kill`.
Please note that the value means how many does a character slay other while being slain by the other.

```suggestion
      "centerDescription": "Ratio de Personnage / Élimination",
```

---

It got changed accordingly to your suggestion